pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -31,6 +31,13 @@ import (
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Namespaced
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="ReadyToUse",type=boolean,JSONPath=`.status.readyToUse`,description="Indicates if a snapshot is ready to be used to restore a volume."
+// +kubebuilder:printcolumn:name="SourcePVC",type=string,JSONPath=`.spec.source.persistentVolumeClaimName`,description="The PVC from where a dynamically taken snapshot will be created."
+// +kubebuilder:printcolumn:name="SourceSnapshotContent",type=string,JSONPath=`.spec.source.volumeSnapshotContentName`,description="The VolumeSnapshotContent from where a snapshot will be created."
+// +kubebuilder:printcolumn:name="RestoreSize",type=string,JSONPath=`.status.restoreSize`,description="Represents the complete size of the snapshot in bytes."
+// +kubebuilder:printcolumn:name="SnapshotClass",type=string,JSONPath=`.spec.volumeSnapshotClassName`,description="The name of the VolumeSnapshotClass requested by the VolumeSnapshot."
+// +kubebuilder:printcolumn:name="SnapshotContent",type=string,JSONPath=`.status.boundVolumeSnapshotContentName`,description="The name of the VolumeSnapshotContent to which this VolumeSnapshot is bound."
+// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.status.creationTime`
```

Comment:
So on line 230 below, `Age` in "// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`" means how long the VolumeSnapshotContent API object is created in the API server, right?  But here "Age" refers to the creationTime of the physical snapshot resource on the storage system.  So they mean different things.  If "Age" (how long the API object is created) is printed for all k8s API objects, maybe we should just do the same for VolumeSnapshot, VolumeSnapshotContent, VolumeSnapshotClass API objects.  In addition, this field from status.creationTime should still be named "CreationTime".

Reply:
I've updated all of the objects to include the `Age`, and also included `CreationTime` on the VolumeSnapshot itself.